### PR TITLE
Null round handling

### PR DESCRIPTION
**Pull request by Qwerty5Uiop**
Resolves https://github.com/blockscout/blockscout/issues/9154

Docs update: https://github.com/blockscout/docs/pull/241

### CODE REVIEW

**Review by vbaranov — CHANGES_REQUESTED** [review]
`mix do ecto.create, ecto.migrate`

```
** (Code.LoadError) could not load /Users/barrakuda/Documents/code/blockscout/apps/explorer/config/dev/filecoin.exs. Reason: enoent
```

File: .github/workflows/publish-docker-image-for-filecoin.yml [hunk 1]
```diff
@@ -36,4 +36,4 @@ jobs:
             CACHE_ADDRESS_WITH_BALANCES_UPDATE_INTERVAL=
             BLOCKSCOUT_VERSION=v${{ env.RELEASE_VERSION }}-beta.+commit.${{ env.SHORT_SHA }}
             RELEASE_VERSION=${{ env.RELEASE_VERSION }}
-            CHAIN_TYPE=polygon_edge
\ No newline at end of file
+            CHAIN_TYPE=filecoin
```
Comment (vbaranov) [inline .github/workflows/publish-docker-image-for-filecoin.yml:39]: It is required also to drop `collated_gas_price_constraint` as it was in `polygon_edge` JSONRPC variant. I added corresponding migration in https://github.com/blockscout/blockscout/pull/9403/commits/1e8a9787f58b83e1ca3d913725ec568bc373509c.

**Review by vbaranov — APPROVED** [review]

File: apps/explorer/priv/filecoin/migrations/20230731130103_modify_collated_gas_price_constraint.exs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+defmodule Explorer.Repo.Migrations.ModifyCollatedGasPriceConstraint do
```
Comment (nikitosing) [inline apps/explorer/priv/filecoin/migrations/20230731130103_modify_collated_gas_price_constraint.exs:1]: Should be Explorer.Repo.Filecoin.Migrations.ModifyCollatedGasPriceConstraint

File: apps/explorer/priv/filecoin/migrations/20231109104957_create_null_round_heights.exs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+defmodule Explorer.Repo.Filecoin.Migrations.CreateNullRoundHeights do
+  use Ecto.Migration
+
+  def change do
+    create table(:null_round_heights, primary_key: false) do
+      add(:height, :integer, primary_key: true)
```
Comment (nikitosing) [inline apps/explorer/priv/filecoin/migrations/20231109104957_create_null_round_heights.exs:6]: I think it should be `bigint` as in `blocks` table

**Review by vbaranov — APPROVED** [review]

**Review by nikitosing — APPROVED** [review]

**Pull request merged**